presto-product-tests/bin/product-tests-suite-5.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+presto-product-tests/bin/run_on_docker.sh \
+    singlenode-hive-impersonation \
+    -g storage_formats,cli,hdfs_impersonation
+
+presto-product-tests/bin/run_on_docker.sh \
+    singlenode-kerberos-hive-impersonation \
+    -g storage_formats,cli,hdfs_impersonation,authorization,hive_file_header
```

Comment:
Any particular reason to include `cli` and `hive_file_header` tests?
(@kokosing do you know why do we run `cli` tests so often?)

Reply:
No particular reason to include those tests. I followed existing hdfs-impersonation tests.